mix.exs

```diff
@@ -1,13 +1,13 @@
 defmodule SpandexPhoenix.MixProject do
   use Mix.Project
 
-  @version "1.0.5"
+  @version "1.0.6"
```

Comment:
Not that it matters in our fork that much, but bumping the minimal Elixir version usually requires a minor version bump.
```suggestion
  @version "1.1.0"
```